plugins/lookup/dsv.py

```diff
@@ -105,11 +105,15 @@
 class LookupModule(LookupBase):
     @staticmethod
     def Client(vault_parameters):
-        return SecretsVault(**vault_parameters)
+        try: 
+            vault = SecretsVault(**vault_parameters)
+            return vault
+        except:
```

Comment:
Better be more specific here:
```suggestion
        except TypeError:
```